src/components/Header/Header.component.jsx

```diff
@@ -1,19 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import InputText from 'ui/InputText';
 import { FaBars, FaUserCircle, FaSun } from 'react-icons/fa';
 import { StyledNavBar } from './Header.styled';
 
-const Header = () => (
-  <StyledNavBar>
-    <StyledNavBar.Left>
-      <FaBars />
-      <InputText />
-    </StyledNavBar.Left>
-    <StyledNavBar.Right>
-      <FaSun />
-      <FaUserCircle />
-    </StyledNavBar.Right>
-  </StyledNavBar>
-);
+const Header = ({ setSearch }) => {
+  const [tempSearch, setTempSearch] = useState();
+  const onSubmit = (ev) => {
+    setSearch(tempSearch);
+    ev.preventDefault();
+  };
+  return (
+    <StyledNavBar>
+      <StyledNavBar.Left>
+        <FaBars />
+        <form onSubmit={onSubmit}>
+          <InputText onChange={setTempSearch} />
+        </form>
```

Comment:
I recommend using [controlled components](https://dmitripavlutin.com/controlled-inputs-using-react-hooks/), it is not necessary because you're solving the issue with uncontrolled components, but, it is a better practice to have controlled components.

```js
<InputText onChange={(event) => setTempSearch(event.target.value)}  value={tempSearch}/>
```